cristin-import/src/test/java/no/unit/nva/cristin/CristinDataGenerator.java

```diff
@@ -199,7 +202,7 @@ public static CristinObject randomObject(String secondaryCategory) {
     public static <T> AwsEventBridgeEvent<FileContentsEvent<JsonNode>> toAwsEvent(T inputData) {
         AwsEventBridgeEvent<FileContentsEvent<JsonNode>> event = new AwsEventBridgeEvent<>();
         JsonNode cristinData = convertToJsonNode(inputData);
-        FileContentsEvent<JsonNode> eventDetail = new FileContentsEvent<>(randomUri(), cristinData);
+        FileContentsEvent<JsonNode> eventDetail = new FileContentsEvent<>(randomUri(), TIMESTAMP, cristinData);
```

Comment:
why hardcoded value here. Also the name of the parameter gives no information. This is an ARBITRARY_TIMESTAMP or shorter SOME_TIMESTAMP. But if you are not using the value anywhere else, why not being a random value?